kge/job/train.py

```diff
@@ -812,8 +820,6 @@ def _prepare(self):
             pin_memory=self.config.get("train.pin_memory"),
         )
 
-        self.is_prepared = True
```

Comment:
Why is anything changed in prepare?

Reply:
Those were redundant, is already set in run()

Reply:
Ok